nfc/libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -208,18 +340,30 @@ phNciNfc_Initialise(
             wStatus =  phNciNfc_LogConnInit(pNciContext);
             if(NFCSTATUS_SUCCESS == wStatus)
             {
-                pNciContext->ResetInfo.ResetTypeReq = eResetType;
-                /* Init payload to display the build number in the init response */
-                pNciContext->tInitInfo.bExtension = 0x00;
-                pNciContext->tInitInfo.bSkipRegisterAllNtfs = 0x00;
-                wStatus = phNciNfc_GenericSequence((void *)pNciContext, NULL, NFCSTATUS_SUCCESS);
-                if(NFCSTATUS_PENDING != wStatus)
+                wStatus = phNciNfc_RegAllNtfs(pNciContext);
```

Comment:
I am moving phNciNfc_RegAllNtfs call before the NCI initialization sequence because i believe this should be done before any NFCC related actions and also to remove bSkipRegisterAllNtfs .

Just unsure, where was the first time phNciNfc_RegAllNtfs  was moved ?